kcp/login.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
```

Comment:
some corresponding instructions would be required in README.md for both the new scripts.

Reply:
Done